google/compute/artman_compute.yaml

```diff
@@ -0,0 +1,24 @@
+common:
+  api_name: compute
+  api_version: v1
+  organization_name: google-cloud
+  gapic_yaml: v1/compute_gapic.yaml
+  # TODO(andrealin): Make discovery_doc a relative path to google/compute
+  discovery_doc: google/compute/v1/compute.v1.json
```

Comment:
Ideally this line would be 
`discovery_doc: google/compute/v1/compute.v1.json`
but the path resolution hasn't been enabled in artman